iati/core/tests/test_schemas.py

```diff
@@ -61,8 +84,8 @@ def test_schema_unmodified_includes(self, schema_initialised):
             Simplify asserts.
 
         """
-        schema = schema_initialised
-        local_element = 'iati-activities'
+        schema = schema_type()
```

Comment:
This is really unclear due to the multiple levels of indirection.

Should be a call to a relevant function in `iati.core.default`.

The same in other cases of this.